### ISSUE TITLE
use of enableConfirms option

### ISSUE THREAD

**Issue by pawelbaranski** [mateodelnorte/servicebus#62]
Hi

I am trying to use enableConfirms options with callback on `send` and I am getting and error that options is not set so I can't use the callback
```js
//...
var bus = require('servicebus').bus({ url: busUrl, enableConfirms: true });

bus.send('test', { xxx: 'yyy'}, function (err) {
  console.log(err);
});
```
output
`[Error: callbacks only supported when created with bus({ enableConfirms:true })]`

am I missing something ?

**Comment by mateodelnorte** [comment]
This is a bug. The check for initialized should happen after the check for
confirms. I can fix and publish an update tonight. Thanks for the find.

**Comment by pawelbaranski** [comment]
I'd really appreciate this, thanks

**Comment by mateodelnorte** [comment]
Hi @pawelbaranski. I've made #63 to fix this issue. I'm on vacation at the moment. Want pull that branch, test to confirm it fixes your issue and then I'll publish tonight? 

`git clone` the repo to another directory, checkout that branch, then npm link it to your project. You'll be able to use it directly.

**Comment by mateodelnorte** [comment]
servicebus now published to version 1.0.20 with this fix included. ;)

**Comment by pawelbaranski** [comment]
nice :+1:  :)
